costmap_2d/cfg/ObstructionPlugin.cfg

```diff
@@ -7,8 +7,13 @@ gen = ParameterGenerator()
 gen.add("enabled", bool_t, 0, "Whether to apply this plugin or not", True)
 gen.add("max_obstacle_height", double_t, 0, "The maximum height of any obstacle to be inserted into the costmap in meters.", 2, 0, 50)
 
-gen.add("cost_scaling_factor", double_t, 0, "A scaling factor to apply to cost values during inflation.", 10, 0, 100)
-gen.add("inflation_radius", double_t, 0, "The radius in meters to which the map inflates obstacle cost values.", 0.55, 0, 50)
+gen.add("distance_threshold", double_t, 0, "How close a reading needs to be to the static layer to count as semi-static.", 0.2, 0, 1)
+
+gen.add("ss_cost_scaling_factor", double_t, 0, "A scaling factor to apply to cost values during inflation.", 10, 0, 100)
```

Comment:
Not static, though.  pseudo_static or semi_static. I will rename these for clarity though.